modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc

```diff
@@ -54,25 +66,37 @@ projects:
       type: git
       location: 'https://github.com/eclipse-che/che-server.git'
 ----
-See the xref:authoring-devfiles-version-1.adoc[Devfile reference].
+For more information about devfiles, see xref:authoring-devfiles-version-2.adoc[Authoring a devfile 2.1.0].
 ====
 
-* To open the workspace, click the btn:[Create & Open] button.
+. Click btn:[Create & Open].
 
+. Wait for the initialized workspace to start and for the IDE to open.
 
 [id="importing-from-the-dashboard-into-an-existing-workspace_{context}"]
-== Importing from the Dashboard into an existing workspace
+== Importing a project into a created workspace
 
-* From the *Dashboard*, select *Workspaces*.
-* Select your workspace by selecting its name. This will link you to the workspace's *Overview* tab.
-* Open *Devfile* tab where you can enter your own YAML configuration.
-* Add the project.
+You can import your project into a created workspace by editing its devfile.
```

Comment:
Perhaps I could replace `its` with either `the project's` or `the workspace's`.

Reply:
I think all three options work. Go for the one you like :)